### PR TITLE
Add configs for simple classic to load odyssey stats

### PR DESCRIPTION
**Pull request by candy02058912**
Related https://github.com/Automattic/dotcom-forge/issues/6213

## Proposed changes:

* Set the correct default value for `jetpack` 
* Set the correct default value for `is_running_in_jetpack_site`
* Add a field for `is_wpcom_simple` for stats to know whether site is a simple site

### Other information:

- [x] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [x] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

See testing instructions in https://github.com/Automattic/wp-calypso/pull/88788

### CODE REVIEW

File: projects/packages/stats-admin/tests/php/test-odyssey-config-data.php [hunk 1]
```diff
@@ -42,6 +42,6 @@ public function test_config_data() {
 		$this->assertArrayHasKey( 'nonce', $data );
 		$this->assertArrayHasKey( 'site_name', $data );
 		$this->assertArrayHasKey( 'intial_state', $data );
-		$this->assertEmpty( $data['features'] );
+		$this->assertNotEmpty( $data['features'] );
```
Comment (DustyReagan) [inline projects/packages/stats-admin/tests/php/test-odyssey-config-data.php]: This should get the tests passing, but maybe it should be more specific? 🤔

Comment (candy02058912) [reply projects/packages/stats-admin/tests/php/test-odyssey-config-data.php]: Made it more specific 👍

**Review by taipeicoder — APPROVED** [review]
LGTM 👍 Some tests are failing but not related to this PR.
See: https://github.com/Automattic/jetpack/pull/36565 and p1711642809295489/1711471112.558629-slack-CDLH4C1UZ for more context.

**Comment by kangzj** [comment]
Sorry I was AFK for the last couple of days, but the changes look good to me! Nice work 👍

**Pull request merged**